x/superfluid/keeper/msg_server.go

```diff
@@ -95,3 +101,40 @@ func (server msgServer) LockAndSuperfluidDelegate(goCtx context.Context, msg *ty
 		ID: lockupRes.ID,
 	}, err
 }
+
+func (server msgServer) UnPoolWhitelistedPool(goCtx context.Context, msg *types.MsgUnPoolWhitelistedPool) (*types.MsgUnPoolWhitelistedPoolResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	if ctx.BlockHeight() < v8constants.UpgradeHeight {
+		return nil, errors.New("message not activated")
+	}
+
+	sender, err := sdk.AccAddressFromBech32(msg.Sender)
+	if err != nil {
+		return nil, err
+	}
+
+	// We get all the lockIDs to unpool
+	lpShareDenom := gammtypes.GetPoolShareDenom(msg.PoolId)
+	minimalDuration := time.Millisecond
```

Comment:
Is the choice of such a small value to make sure that all locked accounts get picked up?

Reply:
I would say yes, any account less then a lock of time.Millisecond could unlock directly anyway